src/components/Header/Header.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import routes from '../../constants/routes.js';
+import RethinkLogo from '../../images/header/RethinkLogo.jpg';
+import './styles.css';
+
+// defuzz logo
```

Comment:
Let's remove these comments and add documentation. I found this article on how to do documentation on react (very similar to Java's documentation)

https://medium.com/@bobjunior542/document-your-react-code-with-jsdoc-best-practices-and-tips-32bf6b92b91f#:~:text=This%20syntax%20is%20similar%20to,for%20writing%20comments%20in%20JavaScript.&text=To%20document%20your%20React%20code%20with%20JSDoc%2C%20you%20simply%20need,followed%20by%20a%20JSDoc%20tag.&text=*%20This%20component%20renders%20a%20greeting%20to%20the%20user.